columnflow/plotting/plot_all.py

```diff
@@ -192,8 +194,8 @@ def plot_all(
     # some default ylim settings based on yscale
     log_y = style_config.get("ax_cfg", {}).get("yscale", "linear") == "log"
 
-    ax_ymin = ax.get_ylim()[1] / 10**4 if log_y else 0.00001
-    ax_ymax = get_position(ax_ymin, ax.get_ylim()[1], factor=1.4, logscale=log_y)
+    ax_ymin = ax.get_ylim()[1] / 10**magnitudes if log_y else 0.0000001
```

Comment:
Why the change for the offset? Is there a specific reason for this?

Reply:
It's not related to the rest of the PR, I think I just had some issue with the offset in my own analysis at some point because signal event yields were on the same order of magnitude as this aribrary offset